api/api.go

```diff
@@ -117,9 +122,11 @@ func New(ctx context.Context, options *Options) (*web.API, error) {
 			filters.NewBasicAuthnFilter(options.Repository),
 			bearerAuthnFilter,
 			secfilters.NewRequiredAuthnFilter(),
-			filters.NewForbiddenLabelOperationsFilter(options.APISettings.ProctedLabels),
+			filters.NewProtectedLabelsFilter(options.APISettings.ProctedLabels),
 			&filters.SelectionCriteria{},
 			&filters.PlatformAwareVisibilityFilter{},
+			filters.NewOIDCTenantCriteriaFilter(options.APISettings.TenantCriteria),
```

Comment:
What if I want another tenant criteria and not this one? Do I leave these and add 2 more filters? What if both tenant filters match the requirements for scoping? Should these be filters that can be added and removed or only 1 tenancy rule can be active and it be added to the service-manager itself and not as a filter?

Reply:
It would be strange to gave to separate tenant matching filters - if you wanted another tenant criteria you would put it in the Tenant Settings